openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/RecognizerFeedbackRepository.java

```diff
@@ -145,7 +143,21 @@ private void validateTagIsAutoApplied(String entityLink, String tagFQN) {
 
       List<TagLabel> tagsToCheck = null;
 
-      if (arrayFieldName != null && fieldName != null) {
+      if (Entity.TABLE.equals(entityType) && Entity.FIELD_COLUMNS.equals(fieldName)) {
```

Comment:
what would happen when we expand this to dashboard data models or topics for example?

Reply:
ok thanks